api/proto/disperser/v2/disperser_v2.proto

```diff
@@ -169,7 +169,10 @@ message GetPaymentStateReply {
   bytes onchain_cumulative_payment = 5;
 }
 
-// GetPaymentStateForAllQuorumsReply contains the payment state of an account.
+// GetPaymentStateForAllQuorumsReply contains the payment state of an account. EigenLabs disperser is the only disperser that allows
+// for ondemand usages, and it will provide the latest on-demand offchain payment records for the request account.
+// Other dispersers will refuse to serve ondemand requests and serve 0 for off-chain on-demand payment usage (`cumulative_payment`). A client using
```

Comment:
what is a "payment records"? Do you mean `period_records`? I don't see payment record anywhere. Also wdym by "serve 0 for off-chain on-demand payment usage"? What is 0? Is it all fields in the Reply that are set to 0? Or only certain ones? Better to talk about the exact fields on this struct to be precise.